src/de/mod.rs

```diff
@@ -947,7 +953,7 @@ mod tests {
         assert_eq!(
             from_str(r#"{ "description": "An ambient temperature sensor" }"#),
             Ok(Property {
-                description: Some("An ambient temperature sensor".to_string()),
+                description: Some("An ambient temperature sensor".into()),
```

Comment:
When I convert to no_std-compatible code, my first pass is simply replacing to_string and to_owned with into.  Then later I might add use ToString at which point those changes become unnecessary.  I personally prefer into since it’s shorter and doesn’t have to deal with Display, but if you prefer I can undo those changes.

Reply:
I don't have a strong preference for one, but I would prefer to have it the same throughout the codebase, so would be nice if you could change it back to `to_string`.